.gitignore

```diff
@@ -20,3 +20,6 @@
 /translations/
 !/translations/.gitkeep
 ###< symfony/translation ###
+
+# Example asset used only directly in the pattern library
+/vendor-extra/LiberoPatternsBundle/src/Resources/public/images/custom-logo.svg
```

Comment:
This should be excluded from the export itself as it’s PL content.

Reply:
Will put this in to the updated styling work for https://github.com/libero/libero/issues/139